drivers/spi/spi_ll_stm32.c

```diff
@@ -338,12 +355,12 @@ static int spi_stm32_configure(struct device *dev,
 	spi_context_cs_configure(&data->ctx);
 
 	LOG_DBG("Installed config %p: freq %uHz (div = %u),"
-		    " mode %u/%u/%u, slave %u",
```

Comment:
Why modifying this?

Reply:
Not by intention. Seems to have happened when SYS_LOG was changed to LOG and I updated.